canonical/vanilla-framework
# test: setup cypress

## Done

- test: setup cypress

This PR will be followed by another one that will [Add Cypress to CI tests](https://github.com/canonical/vanilla-framework/issues/4541)

Fixes: https://github.com/canonical/vanilla-framework/issues/4538

## QA

- Run `yarn cypress:open` or `yarn cypress:run` and verify the example accordion test passes

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

[if relevant, include a screenshot or screen capture]
